### PR TITLE
New logos

### PR DESCRIPTION
**Pull request by suprMax**

### CODE REVIEW

File: packages/components/src/logo/index.jsx [hunk 1]
```diff
@@ -3,36 +3,41 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import styles from './index.module.scss';
 
+import LogoText from '../logo_text';
+
+import { ReactComponent as GHLogo } from './images/logo.svg';
```
Comment (peterhass) [inline packages/components/src/logo/index.jsx:8]: `CompanyLogo`

File: packages/components/src/logo/index.jsx [hunk 2]
```diff
@@ -3,36 +3,41 @@ import PropTypes from 'prop-types';
...
+import { ReactComponent as GHChristmasLogo } from './images/logo-christmas.svg';
```
Comment (peterhass) [inline packages/components/src/logo/index.jsx:9]: `CompanyChristmasLogo`

File: packages/components/src/logo/index.jsx [hunk 3]
```diff
@@ -3,36 +3,41 @@ import PropTypes from 'prop-types';
...
+  const content = <><LogoImage className={styles.image} />{text}</>;
 
   if (to) return <a {...cleanProps} className={className} href={to}>{content}</a>;
-
   return <span {...cleanProps} className={className}>{content}</span>;
 };
 
 Logo.defaultProps = {
   compact: false,
-  localeName: 'de_de',
```
Comment (peterhass) [inline packages/components/src/logo/index.jsx:30]: that's a breaking change imo

Comment (suprMax) [reply packages/components/src/logo/index.jsx:30]: yes, but i checked and these logos are not in use anywhere, we have to upgrade manually anyway

Comment (peterhass) [reply packages/components/src/logo/index.jsx:30]: it's still a breaking change

**Pull request merged**